webservices/rulemaking_docs/__init__.py

```diff
@@ -0,0 +1,102 @@
+import logging
+import sys
+
+from .rulemaking import load_rulemaking
+
+
+logging.basicConfig(level=logging.INFO, stream=sys.stdout)
+logger = logging.getLogger("elasticsearch")
+logger.setLevel("WARN")
+logger = logging.getLogger("botocore")
+logger.setLevel("WARN")
+
+
+def reload_all_data_by_index(index_name=None):
+    """
+    - Reload all legal data by specify 'XXXX_INDEX' (it takes 15mins ~ 2+ hours) without downtime.
+    'INDEX_DICT' description:
+    1) CASE_INDEX includes DOCUMENT_TYPE=('murs','adrs','admin_fines')
+    'murs' means current mur only.
+    2) AO_INDEX includes DOCUMENT_TYPE=('advisory_opinions','statutes')
+    3) ARCH_MUR_INDEX includes DOCUMENT_TYPE=('murs'), archived mur only
+
+    - How to call task command:
+    a) cf run-task api --command "python cli.py reload_all_data_by_index case_index" -m 4G --name reload_all_data_case
+    b) cf run-task api --command "python cli.py reload_all_data_by_index ao_index" -m 4G --name reload_all_data_ao
+    c) cf run-task api --command "python cli.py reload_all_data_by_index arch_mur_index" -m 4G
+    --name reload_all_data_arch_mur
+    """
+
+    load_rulemaking()
+    # if index_name in INDEX_DICT.keys():
```

Comment:
@fec-jli Can you remove the commented out code from L31-42 if its no longer needed?